src/components/ArticleManager.jsx

```diff
@@ -0,0 +1,58 @@
+// header
+// nav bar
+// article card
+// comment manager > comment card
+// votes
+
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+import { useEffect } from "react";
+
+export default function ArticleManager() {
+  const [selectedArticle, setSelectedArticle] = useState();
+  const { article_id } = useParams();
+
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetch(`https://news-app-n80t.onrender.com/api/articles/${article_id}`)
+      .then((data) => {
+        return data.json();
+      })
```

Comment:
I recommend that this becomes a reusable function instead of the whole `fetch` call just here. Recall last week we had an `api.js` file in ` utils` folder.